hawkular-inventory-rest-api/pom.xml

```diff
@@ -59,6 +59,18 @@
     </dependency>
 
     <dependency>
+      <groupId>org.hawkular.inventory</groupId>
+      <artifactId>hawkular-inventory-security-spi</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <dependency>
```

Comment:
I was talking about the hard compile dependency on `hawkular-inventory-security-permissive`. The dependency on the security SPI is of course alright.